pandas/_libs/tslibs/parsing.pyx

```diff
@@ -298,6 +298,14 @@ def parse_datetime_string(
     if dt is not None:
         return dt
 
+    # Handling special case strings today & now
+    if date_string == "now":
+        dt = datetime.now()
+    elif date_string == "today":
+        dt = datetime.today()
+    if dt is not None:
```

Comment:
Nit: Can just add `return datetime...` in the if clauses instead.

Reply:
Sure @mroeschke , have updated the PR